Resources/Textures/Nuclear14/Objects/Consumable/Drinks/nuka_cola.rsi/meta.json

```diff
@@ -0,0 +1,15 @@
+{
+  "version": 1,
```

Comment:
needs copyright info to pass linter. Can't find it in Desert rose or I'd add it myself.